### PR TITLE
gh-71616: Add note to warn against general translation of saxutils.escape()

### PR DESCRIPTION
**Pull request by slateny**
#71616

Co-authored-by: Xiang Zhang <angwerzx@126.com>

https://docs.python.org/3/library/xml.sax.utils.html

### CODE REVIEW

**Review by DanielNoord — APPROVED** [review]

File: Doc/library/xml.sax.utils.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@ or as base classes.
    replaced with its corresponding value.  The characters ``'&'``, ``'<'`` and
    ``'>'`` are always escaped, even if *entities* is provided.
 
+   .. note::
+
+      This function should only be used to escape characters that are
+      problematic to include directly in XML. Do not use this function as a general
```
Comment (nanjekyejoannah) [inline Doc/library/xml.sax.utils.rst]: This wording doesnt seem too ideal for me, can we use serhey's `that can't be used directly in XML ...`.

Comment (slateny) [inline Doc/library/xml.sax.utils.rst]:
```suggestion
      This function should only be used to escape characters that
      can't be used directly in XML. Do not use this function as a general
```

**Review by nanjekyejoannah — APPROVED** [review]

**Review by nanjekyejoannah — COMMENTED** [review]
LGTM

**Comment by nanjekyejoannah** [comment]
Thanks @slateny

**Pull request merged**